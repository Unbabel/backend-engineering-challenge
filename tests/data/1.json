{"timestamp": "2020-01-01 01:11:08.509654","translation_id": "6aa5b2f39f7254a75aa5","source_language": "en","target_language": "fr","client_name": "easyjet","event_name": "translation_delivered","nr_words": 30, "duration": 200}
{"timestamp": "2020-01-01 02:11:19.903159","translation_id": "6aa5b2f39f7254a75aa4","source_language": "en","target_language": "fr","client_name": "easyjet","event_name": "translation_delivered","nr_words": 30, "duration": 300}
