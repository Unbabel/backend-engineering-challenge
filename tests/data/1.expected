{"date": "2020-01-01 01:11:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:12:00", "average_delivery_time": 200.0}
{"date": "2020-01-01 01:13:00", "average_delivery_time": 200.0}
{"date": "2020-01-01 01:14:00", "average_delivery_time": 200.0}
{"date": "2020-01-01 01:15:00", "average_delivery_time": 200.0}
{"date": "2020-01-01 01:16:00", "average_delivery_time": 200.0}
{"date": "2020-01-01 01:17:00", "average_delivery_time": 200.0}
{"date": "2020-01-01 01:18:00", "average_delivery_time": 200.0}
{"date": "2020-01-01 01:19:00", "average_delivery_time": 200.0}
{"date": "2020-01-01 01:20:00", "average_delivery_time": 200.0}
{"date": "2020-01-01 01:21:00", "average_delivery_time": 200.0}
{"date": "2020-01-01 01:22:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:23:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:24:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:25:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:26:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:27:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:28:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:29:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:30:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:31:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:32:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:33:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:34:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:35:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:36:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:37:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:38:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:39:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:40:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:41:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:42:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:43:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:44:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:45:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:46:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:47:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:48:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:49:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:50:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:51:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:52:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:53:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:54:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:55:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:56:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:57:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:58:00", "average_delivery_time": 0}
{"date": "2020-01-01 01:59:00", "average_delivery_time": 0}
{"date": "2020-01-01 02:00:00", "average_delivery_time": 0}
{"date": "2020-01-01 02:01:00", "average_delivery_time": 0}
{"date": "2020-01-01 02:02:00", "average_delivery_time": 0}
{"date": "2020-01-01 02:03:00", "average_delivery_time": 0}
{"date": "2020-01-01 02:04:00", "average_delivery_time": 0}
{"date": "2020-01-01 02:05:00", "average_delivery_time": 0}
{"date": "2020-01-01 02:06:00", "average_delivery_time": 0}
{"date": "2020-01-01 02:07:00", "average_delivery_time": 0}
{"date": "2020-01-01 02:08:00", "average_delivery_time": 0}
{"date": "2020-01-01 02:09:00", "average_delivery_time": 0}
{"date": "2020-01-01 02:10:00", "average_delivery_time": 0}
{"date": "2020-01-01 02:11:00", "average_delivery_time": 0}
{"date": "2020-01-01 02:12:00", "average_delivery_time": 300.0}
